src/sentry/tasks/store.py

```diff
@@ -348,6 +357,15 @@ def _continue_to_process_event():
                     has_changed = True
                     break
 
+    if submit_realtime_metrics:
+        symbolication_duration = time() - symbolication_start_time
+        with sentry_sdk.start_span(
+            op="tasks.store.symbolicate_event.low_priority.metrics.histogram"
+        ):
+            realtime_metrics.increment_project_duration_counter(
+                project_id, symbolication_start_time, symbolication_duration
+            )
+
```

Comment:
any reason why both realtime metrics couldn't be reported together here?

also, echoing what @beezz mentioned in the other PR: it would be a good idea to ensure that any exceptions thrown by this don't interrupt event processing.